client/app/components/packages/rwcds-form/proposed-project-actions-editor.js

```diff
@@ -9,32 +10,34 @@ export default class ProposedActionsComponent extends Component {
     'ZA', 'ZS', 'SD', 'SA', 'RA', 'RS',
   ];
 
+  get zrTypeLabel() {
+    const zrTypeCode = this.args.zrForm.saveableChanges.dcpZoningresolutiontype;
+    const zrLookup = AFFECTED_ZONING_RESOLUTION_ACTION_OPTIONSET;
+
+    return Object.values(zrLookup).findBy('code', zrTypeCode).label;
```

Comment:
Here, we can use the optionset helper to achieve the same thing. We don't need `const zrLookup` here, and the optionset helper, after the your awesome work registering the new optionset with it, will perform the `Object.values(zrLookup).findBy( ...` for us! :D 

```
return optionset(['affectedZoningResolution', 'actions', 'label', zrTypeCode]);
```

I wrote up a readme on how to use the optionset helper here:
`client/app/helpers/helpers.md`

Reply:
amazing! This readme is great :) Very fond of this helper you built